src/app/screens/HomeScreen/constants.ts

```diff
@@ -4,7 +4,7 @@ export const DATA: Array<Book> = [
   {
     id: 1,
     author: 'Emmie Thiel',
-    title: 'Ring of Bright Water',
+    title: 'Ring of Bright Water Ring of Bright Water Ring of',
```

Comment:
⚠️

Reply:
I added this to have a case where the title is long enough to have 2 lines, I would like to leave it since we are already using mocked data

Reply:
I will change it for a more realistic text tho :P